app/models/favorite.rb

```diff
@@ -6,11 +6,17 @@ def initialize(initial_contents)
   end
 
   def add_pet(pet)
-    @contents << pet
+    @contents << pet.id
```

Comment:
I see we've limited what goes inside our @contents array.  Isn't this where we also gathered our pet image?

Reply:
I created a method that takes the pets id and and calls the pet object from it. Kind of a backwards work around (back to Kansas), but it works. I tried all kinds of variables to get the incrementation and decrementation to function, but the only way that would work was with the .id

Reply:
I thought about this some more and realized that we can load up that information from active record so long as we have the id.  Nice clean up!